src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/imports/CustomImportOrderCheck.xml

```diff
@@ -107,9 +111,8 @@
          <properties>
             <property default-value=""
                       name="customImportOrderRules"
-                      type="java.lang.String">
-               <description>Specify format of order declaration
- customizing by user.</description>
+                      type="java.lang.String[]">
+               <description>Specify list of import groups.</description>
```

Comment:
`Specify list of import rules`  may be ?
`Specify list of import group rules` ?

Reply:
@rnveach, please suggest how to name it. It use to be bad, becomes a bit better, but may be there is slightly better way to word it.